app/src/features/reports/CompareMapNavReport.tsx

```diff
@@ -7,115 +7,113 @@ import useDatasetStore from "../../utils/useDatasetStore";
+      )}
+    </>
+  );
 }
 
 function Map(props: {
-  fipsGeo: number;
+  fipsGeo: string;
   data: Record<string, any>[];
-  updateGeoCallback: Function;
+  updateGeoCallback: (message: string) => void;
 }) {
-  function getStateGeoFromPropFips() {
-    return props.fipsGeo === 0
-      ? undefined
-      : { fips: props.fipsGeo, name: STATE_FIPS_MAP[props.fipsGeo] };
-  }
-
-  const [state, setState] = useState<Geo | undefined>(
-    getStateGeoFromPropFips()
-  );
-  const [county, setCounty] = useState<Geo | undefined>();
+  const [countyFips, setCountyFips] = useState<string>();
+  const [countyName, setCountyName] = useState<string>();
 
   useEffect(() => {
-    props.updateGeoCallback(state ? state.fips : 0);
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [state]);
-
-  useEffect(() => {
-    setState(getStateGeoFromPropFips());
-    setCounty(undefined);
+    setCountyFips(undefined);
+    setCountyName(undefined);
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [props.fipsGeo]);
 
   const signalListeners: any = {
     click: (...args: any) => {
-      // Is this a county or a state??
       const clickedData = args[1];
-      console.log(clickedData);
       if (clickedData.id.length === 2) {
-        setState({ fips: clickedData.id, name: clickedData.properties.name });
+        // Updating the state
+        props.updateGeoCallback(clickedData.id);
       } else {
-        setCounty({ fips: clickedData.id, name: clickedData.properties.name });
+        // Updating the county
+        setCountyFips(clickedData.id);
+        setCountyName(clickedData.properties.name);
       }
     },
   };
 
   let dataset =
-    state === undefined
+    props.fipsGeo === USA_FIPS
       ? props.data
-      : props.data.filter((r) => r.state_fips_code === state.fips.toString());
+      : props.data.filter((r) => r.state_fips_code === props.fipsGeo);
 
+  // TODO - make the mouse turn into a pointer when you hover over
   return (
     <div>
       <Breadcrumbs aria-label="breadcrumb">
-        {state && (
-          <Link
-            color="inherit"
-            onClick={() => {
-              setState(undefined);
-              setCounty(undefined);
-            }}
-          >
-            United States
-          </Link>
-        )}
-        {!state && <Typography color="textPrimary">United States</Typography>}
-        {state && county && (
-          <Link
-            color="inherit"
+        <GeographyBreadcrumb
+          text={USA_DISPLAY_NAME}
+          isClickable={props.fipsGeo !== USA_FIPS}
+          onClick={() => {
+            props.updateGeoCallback(USA_FIPS);
+            setCountyFips(undefined);
+            setCountyName(undefined);
+          }}
+        />
+        {props.fipsGeo !== USA_FIPS && (
+          <GeographyBreadcrumb
+            text={STATE_FIPS_MAP[props.fipsGeo]}
+            isClickable={!(!props.fipsGeo || !countyFips)}
```

Comment:
I think you can drop the props.fipsGeo part since that's required on line 81, and then this just becomes !!countyFips, right?

Reply:
oh ya good catch!!